.github/workflows/deploy.yml

```diff
@@ -11,17 +11,17 @@ jobs:
       - name: Checkout source
         uses: actions/checkout@v2
 
-      - name: Set up Python 3.8
+      - name: Set up Python 3.9
         uses: actions/setup-python@v1
         with:
-          python-version: 3.8
+          python-version: 3.9
 
```

Comment:
We could set an environment variable at the top, and use it elsewhere https://docs.github.com/en/actions/learn-github-actions/environment-variables
```
env:
    python_version: "3.9"
```

Reply:
I've added an issue for this: #162